.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -221,52 +224,34 @@ Try editing this function on your own! *(Don't forget to save when you make chan
 
-## **:pencil: Task: Create and deploy an HTTP trigger Azure Function that outputs the current time and date.**
-
-:exclamation: *Don't forget to `git pull` before making any changes to your local repo!!*
-
-Example output:
-```
-Hello! The current time is: 17 : 41 : 34 PM
-```
-
-This time we'll try developing locally. Follow the directions in this [link](https://docs.microsoft.com/en-us/azure/azure-functions/functions-develop-local) to set up and deploy your first Azure Function right in your local machine.
-
+## Request Parameters
 <details>
-<summary>:question: Not sure where to start?</summary>
+<summary>:question: What is a request parameter?</summary>
     </br>
-    
-We need to edit the function so that it returns the current time.
 
-1. First start by creating a variable that gets the current time (try googling "how to get current time in Javascript").
-2. Try to test your function to make sure that it is returning the time in the right format (and timezone).
-3. Once you get the time printing, you can remove any extra code related to getting the parameters since we don't need that for this task. Make sure your output matches the example output with accurate current time.
+Request parameters are a way for an HTTP request to take in information! They are pretty much identical in purpose to why you would want a parameter for a function in a coding language. In this instance, we will need parameters for the username and password.
+<br><br/>
+</details>
 
 <details>
-<summary>:question: Still stuck? Check out the code below:</summary>
-    </br>
-
-Javascript has a built in [Date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) that evaluates to the time in milliseconds since 1 January 1970 UTC (keep the timezone in mind).
+<summary>:question: How do I create a request parameter?</summary>
+<br></br>
+Look at the code for a basic HTTP Function:
 
-First you want to create an instance of the Date object:
 ```javascript
-var currentdate = new Date(); 
+module.exports = function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+    context.done();
+};
 ```
-
-Next, you can use the [Instance Methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#instance_methods) in the linked documentation to use methods that calculate the current hours, minutes, and seconds to get the current time.
-
+Request parameters are a property of the `req` or request parameter of the module. The request has a `query` object that stores all of the parameters passed in. You would access a parameter by calling on the query like this:
 ```javascript
-var currentime = currentdate.getHours() + ":" 
-    + currentdate.getMinutes() + ":" 
-    + currentdate.getSeconds();
+<property name> = req.query.<your property here>;
```

Comment:
Give an example like if someone sent in a parameter in the URL with `?password=hi`, how would you receive that value?

Reply:
Specify that their parameter *must* be named password